src/resources/extensions/quarto/video/video.lua

```diff
@@ -116,10 +127,22 @@ local vimeoBuilder = function(params)
   if not (params and params.src) then return nil end
 
   local VIMEO_STANDARD = 'https://vimeo.com/'
-  local match = checkEndMatch(params.src, VIMEO_STANDARD)
+  local match = checkMatchStart(params.src, VIMEO_STANDARD)
   if not match then return nil end
 
-  params.src = 'https://player.vimeo.com/video/' .. match
+  local internalMatch = string.match(params.src, 'vimeo.com/(.-)?share=copy')
```

Comment:
I think you want to make the `?share=copy` part optional. This is query parameter for url string, but the url that can be used in shortcode could be `https://vimeo.com/548291210/fdcc90d662`

I believe this is valid usage and should be dealt the same
```` 
{{ vimeo https://vimeo.com/548291210/fdcc90d662 }}
````

I understand we get the links with query parameter when clicking on the share button, but it seems fair to support the version without. Otherwise, we can wait for someone to use that form and report to avoid unnecessary precaution support.

Reply:
Thanks for suggesting this, I've updated the PR.  👍